### ISSUE TITLE
Keybindings for creating/closing a panel

### ISSUE THREAD

**Issue by mrpandey** [mbnuqw/sidebery#534]
## Description
In the keybindings list, there is no option for creating a new panel or closing the current one.
It would be very useful for people who interact mostly through the keyboard.

**Comment by rafaelliu** [comment]
Yes! It feels SideBerry has a more static view of panels but some of us use panels as containers for quick, contextual work that we may switch over to / close as needed
